db/migrate/20160126121119_add_requires_retrial_details_flag_to_case_types.rb

```diff
@@ -1,9 +1,8 @@
 class AddRequiresRetrialDetailsFlagToCaseTypes < ActiveRecord::Migration
   def up
     add_column :case_types, :requires_retrial_dates, :boolean, default: false
-
-    # migrate data
-    CaseType.find_by(name: 'Retrial').update_column(:requires_retrial_dates,true) rescue nil
+    retrial = CaseType.find_by_sql("SELECT * FROM case_types WHERE name = 'Retrial'").first
```

Comment:
what is the purpose of this - its an old migration and does not appear to change the result